### PR TITLE
Add support for formatting dates

### PR DESCRIPTION
**Pull request by megawac**
Another option (like in toSrc) is to show `Date(<the time of creation in ms>)`

### CODE REVIEW

File: src/pretty-format.js [hunk 1]
```diff
@@ -85,6 +85,20 @@ Type.Circular = new Type({
...
  * @public
+ * @class Date
+ * @extends Type
+ * @memberOf Type
+ */
+Type.Date = new Type({
+  test: _.isDate,
+
+  print(val) {
+    return Date.prototype.toISOString.call(val);
```
Comment (megawac) [inline src/pretty-format.js]: Note IE9+

Comment (thejameskyle) [inline src/pretty-format.js]: I'm fine with that

**Comment by thejameskyle** [comment]
:+1: Cool

**Comment by thejameskyle** [comment]
Released in 1.1.0

**Pull request merged**